### PR TITLE
Load from repo

### PR DESCRIPTION
**Pull request by edwardcallahan**
1.1: Future[PR]

### CODE REVIEW

File: src/main/play-doc/operation/Install.md [hunk 1]
```diff
@@ -310,10 +310,10 @@ _Execute the step in this section only on the `172.17.0.1` machine. We could als
 
 These instructions for loading and running the ConductR-HAProxy bundle require the [[CLI|CLI]] to be installed. Continue with the next step once [[ConductR CLI|CLI]] is installed.
 
-Load the ConductR-HAProxy bundle.
+Load the ConductR-HAProxy bundle from the [bundles repo](https://bintray.com/typesafe/bundle).
 
 ```bash
-[172.17.0.1]$ conduct load file:/usr/share/conductr/extra/conductr-haproxy-{version}-{digest}.zip
+[172.17.0.1]$ conduct load conductr-haproxy
```
Comment (huntc) [inline src/main/play-doc/operation/Install.md:316]: yay!

**Review by huntc — APPROVED** [review]
This is such a nice user experience! We can update 1.1 also right...?

**Comment by edwardcallahan** [comment]
We still use native packages for 1.1, not bundles.

**Comment by huntc** [comment]
> We still use native packages for 1.1, not bundles.

A splendid point.

**Pull request merged**